app/src/main/res/layout/fragment_suggested_edits_tasks.xml

```diff
@@ -1,88 +1,173 @@
-    xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    android:background="?android:attr/windowBackground"
+    android:background="?attr/main_toolbar_color"
     android:orientation="vertical">
 
-    <FrameLayout
-        android:id="@+id/userContributionsButton"
+    <org.wikipedia.views.SuggestedEditsDisabledStatesView
+        android:id="@+id/disabledStatesView"
         android:layout_width="match_parent"
-        android:layout_height="72dp"
-        android:background="?attr/main_toolbar_color"
-        android:clickable="true"
-        android:elevation="4dp"
-        android:focusable="true"
-        android:foreground="?attr/selectableItemBackgroundBorderless">
+        android:layout_height="match_parent"
+        android:paddingStart="@dimen/suggested_edits_screen_margin"
+        android:paddingEnd="@dimen/suggested_edits_screen_margin"
+        android:visibility="gone" />
+
+    <androidx.constraintlayout.widget.ConstraintLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:paddingStart="@dimen/suggested_edits_screen_margin"
+        android:paddingEnd="@dimen/suggested_edits_screen_margin"
+        android:layout_marginBottom="@dimen/suggested_edits_stats_bottom_margin">
+
+        <org.wikipedia.views.ImageTitleDescriptionView
+            android:id="@+id/contributionsStatsView"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toTopOf="parent" />
+
+        <org.wikipedia.views.ImageTitleDescriptionView
+            android:id="@+id/editStreakStatsView"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintHorizontal_chainStyle="spread"
+            app:layout_constraintStart_toStartOf="@+id/editQualityStatsView"
+            app:layout_constraintTop_toTopOf="parent" />
+
+        <org.wikipedia.views.ImageTitleDescriptionView
+            android:id="@+id/pageViewStatsView"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            app:layout_constraintBottom_toBottomOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@+id/contributionsStatsView" />
+
+        <org.wikipedia.views.ImageTitleDescriptionView
+            android:id="@+id/editQualityStatsView"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            app:layout_constraintBottom_toBottomOf="parent"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintTop_toBottomOf="@+id/editStreakStatsView" />
```

Comment:
The gap between the left column and the right column is too wide for me. I am using a Pixel 3 XL emulator.

Reply:
Yes I saw this too ... need to ask this question to Robin... I have not centered the views, just set a start and end margin and the margin is different for small and large devices.. so will ask him what to do in cases where the screen is big, but not big enough to trigger a sw600 layout.. nice catch!